src/renderer/widgets/CreateWallet/model/confirm-model.ts

```diff
@@ -0,0 +1,75 @@
+import { createEvent, combine, restore } from 'effector';
+
+import { networkModel } from '@entities/network';
+import { walletModel, walletUtils } from '@entities/wallet';
+import { AddMultisigStore } from '../lib/types';
+
+const formInitiated = createEvent<AddMultisigStore>();
+const formSubmitted = createEvent();
+
+const $confirmStore = restore(formInitiated, null).reset(formSubmitted);
+
+const $api = combine(
+  {
+    apis: networkModel.$apis,
+    store: $confirmStore,
+  },
+  ({ apis, store }) => {
+    return store?.chain ? apis[store.chain.chainId] : undefined;
+  },
+  { skipVoid: false },
+);
+
+const $initiatorWallet = combine(
+  {
+    store: $confirmStore,
+    wallets: walletModel.$wallets,
+  },
+  ({ store, wallets }) => {
+    if (!store) return null;
+
+    return walletUtils.getWalletById(wallets, store.signer.walletId);
+  },
+  { skipVoid: false },
+);
+
+// const $proxiedWallet = combine(
```

Comment:
Remove commented code or add comment when we should uncomment it

Reply:
I ended up deleting this :+1: